doc/faq/howto_faq.rst

```diff
@@ -342,7 +341,7 @@ Control the depth of plot elements
 ----------------------------------
 
 
-Within an axes, the order that the various lines, markers, text,
+Within axes, the order that the various lines, markers, text,
```

Comment:
I like the original phrasing better here, as it points towards the singular form, but perhaps "an axes instance" might be better?